pantheon-bundle/frontend/src/app/moduleDisplay.tsx

```diff
@@ -175,11 +175,11 @@ class ModuleDisplay extends Component<any, any, any> {
     private fetchModuleDetails = async(data) => {
         await this.getVariantParam()
         this.setState({
-            modulePath: data.location.pathname,
-            releasePath: "/content" + data.location.pathname + ".preview?variant=" + this.state.variant
+            modulePath: data.location.pathname.substring(7),
```

Comment:
I am not aware about the context here, but why are we chaning module path, can we do it in a generic way ?

Reply:
currently our routes are configured as
   * the route "/search" loads the Search component
   * the route "/product" loads the Product component
   * the route "/products" loads the  ProductListing component
   * the route "/git" loads the GitImport component
   * the route "/login" loads the Login component 
   * Loads ModuleDisplay component if path has :data value
Because of the last condition, today we can load assembly details on module detail page. This route is not ideal if we want to have a separate assembly detail page. That's why i updated the route logic to allow assembly detail page to be loaded on its own.